prow/monitoring/prow_prometheusrule.yaml

```diff
@@ -0,0 +1,118 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  labels:
+    prometheus: prow
+    role: alert-rules
+  name: prow
```

Comment:
Can we implement this before adding the Services and ServiceMonitors?

Reply:
I think so.
Worse case: it will send alert requests to alertManager which is not with correct webkook url yet.